### ISSUE TITLE
XF2 Update

### ISSUE THREAD

**Issue by thePeterG** [iversia/xenforo-faq-manager#7]
Any chance this will be updated for Xenforo 2?

**Comment by iversia** [comment]
Yep - essentially when get around to upgrading my own Xenforo forums to 2.

**Comment by TheFlash787** [comment]
Hey, I'd really like to see this become a reality. Especially with many using XF2 nowadays, XF1 can become limiting in 2018. This would be fantastic if it's made possible. Thanks!
